assets/css/_sass/_home.scss

```diff
@@ -56,25 +56,34 @@ html, body {
 .home-landing__big-title {
 	position: relative;
 	height: calc(100vh + 1px);
+	min-height: 600px;
 	width: 100%;
-	padding: 0 4.167%;
+	// padding: 0 4.167%;
```

Comment:
It looks like this `//` wasn't there before—why are we commenting this out?

Reply:
This is just a CSS comment showing what was there before and what I've changed, because when changing the HTML the CSS needed to change too.

Reply:
Thank you! A number of my comments are just me having trouble reading GitHub's way of displaying changes—I appreciate the explanation!